packages/make-hardener/rollup.config.js

```diff
@@ -1,16 +1,37 @@
+import resolve from '@rollup/plugin-node-resolve';
+import { terser } from 'rollup-plugin-terser';
+
 export default [
   {
     input: 'src/main.js',
     output: [
       {
-        file: 'dist/make-hardener.umd.js',
-        format: 'umd',
-        name: 'makeHardener',
+        file: 'dist/make-hardener.mjs',
+        format: 'esm',
       },
       {
-        file: 'dist/make-hardener.cjs.js',
+        file: 'dist/make-hardener.cjs',
         format: 'cjs',
       },
     ],
+    plugins: [resolve()],
+  },
+  {
+    input: 'src/main.js',
+    output: {
+      file: 'dist/make-hardener.umd.js',
+      format: 'umd',
+      name: 'make-hardener',
```

Comment:
This is the global that is introduced by the UMD package.  It should be a regular identifier.

```suggestion
      name: 'makeHardener',
```